pennylane_lightning/core/_serialize.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         self.use_csingle = use_csingle
         self.device_name = device_name
         self.split_obs = split_obs
-        if device_name == "lightning.qubit":
+        if device_name in ("lightning.qubit", "lightning.qubit2"):
```

Comment:
That is a nice suggestion, but I'm afraid that if in the future someone creates a `lightning.qubit3`, it would also satisfy this condition. I prefer to have explicit full names here. In my PR migrating lightning.qubit2 to lightning.qubit I will probably change it to what we had before anyway. For now, I think we can keep it.

Reply:
Fair point! :+1: